includes/definitions/discovery/gaia.yaml

```diff
@@ -64,7 +65,7 @@ modules:
                     value: haState
                     states:
                         - { value: 1, descr: active , graph: 3, generic: 0 }
-                        - { value: 2, descr: standby, graph: 2, generic: 0 }
+                        - { value: 2, descr: standby, graph: 2, generic: 3 }
                         - { value: 3, descr: 'active attention', graph: 1, generic: 1 }
```

Comment:
I was testing and maybe I found a issue here, I think the discovery does not treat case sensitive

-bash-4.2$ snmpwalk -v2c -c public 10.10.10.20 -m /opt/librenms/mibs/checkpoint/CHECKPOINT-MIB haState
CHECKPOINT-MIB::haState.0 = STRING: Active Attention
-bash-4.2$

![image](https://user-images.githubusercontent.com/79482451/117225831-a7416c80-ade9-11eb-8152-71e6a1c836e8.png)

After I've changed to capital a it worked:
                        
![image](https://user-images.githubusercontent.com/79482451/117225958-f2f41600-ade9-11eb-819c-506293b1ad2f.png)